config/default/core.entity_form_display.user.user.default.yml

```diff
@@ -3,7 +3,7 @@ langcode: en
 status: true
 dependencies:
   config:
-    - field.field.user.user.field_foia_office
+    - field.field.user.user.field_agency
```

Comment:
We need to prevent users from being able to edit their own agency on their /user/uid/edit pages

Reply:
This is working now!